src/syscheck/syscheck.fpp

```diff
@@ -53,7 +53,7 @@ program syscheck
 
     @:ACC(integer(acc_device_kind) :: devtype)
     @:ACC(integer :: i, num_devices)
-    @:ACC(real(kind(0d0)), allocatable, dimension(:) :: arr)
+    @:ACC(real(kind(0e0_wp)), allocatable, dimension(:) :: arr)
```

Comment:
This appears to be the issue, looks like my sed command changed this to wp too since I ran it for the entire src directory.

Reply:
This is why it's failing the GPUs.

Reply:
you can remove all changes to `syscheck` since it not part of the main codebase. It is  only used to as a pre-check to see if the rest of MFC will run. We can amend `syscheck` to also use different precisions, but I don't think this is necessary for the PR.

Reply:
Yes I fixed it.